File: go/spec/check_service.go
package spec

import (
	"cloud.google.com/go/civil"
	"github.com/google/uuid"
	"github.com/shopspring/decimal"
)

type CheckService struct{}

func (service *CheckService) CheckQuery(
	pString string,
	pStringOpt *string,
	pStringArray []string,
	pDate civil.Date,
	pDateArray []civil.Date,
	pDatetime civil.DateTime,
	pInt int,
	pLong int64,
	pDecimal decimal.Decimal,
	pEnum Choice,
	pStringDefaulted string) (*CheckQueryResponse, error) {

	return &CheckQueryResponse{Ok: &Empty}, nil
}

func (service *CheckService) CheckUrlParams(
	intUrl int64,
	stringUrl string,
	floatUrl float32,
	boolUrl bool,
	uuidUrl uuid.UUID,
	decimalUrl decimal.Decimal,
	dateUrl civil.Date) (*CheckUrlParamsResponse, error) {

	return &CheckUrlParamsResponse{Ok: &Empty}, nil
}

func (service *CheckService) CheckForbidden() (*CheckForbiddenResponse, error) {
	return &CheckForbiddenResponse{Forbidden: &Empty}, nil
}

File: java-spring/src/main/java/test_service/implementations/CheckService.java
package test_service.implementations;

import org.springframework.stereotype.Service;
import test_service.models.*;
import test_service.services.check.*;

import java.math.BigDecimal;
import java.time.*;
import java.util.UUID;

@Service
public class CheckService implements ICheckService {

	@Override
	public CheckEmptyResponse checkEmpty() {
		return new CheckEmptyResponse(new Empty());
	}

	@Override
	public CheckQueryResponse checkQuery(String pString, String pStringOpt, String[] pStringArray, LocalDate pDate, LocalDate[] pDateArray, LocalDateTime pDatetime, int pInt, long pLong, BigDecimal pDecimal, Choice pEnum, String pStringDefaulted) {
		return new CheckQueryResponse(new Empty());
	}

	@Override
	public CheckUrlParamsResponse checkUrlParams(long intUrl, String stringUrl, float floatUrl, boolean boolUrl, UUID uuidUrl, BigDecimal decimalUrl, LocalDate dateUrl, Choice enumUrl) {
		return new CheckUrlParamsResponse(new Empty());
	}

	@Override
	public CheckUnionResponse checkForbidden() {
		return new CheckForbiddenResponse(new Empty());
	}
}
